tests/test_studio.py

```diff
@@ -243,3 +243,46 @@ def test_post_to_studio_skip_if_no_token(tmp_dir, mocker, monkeypatch):
         live.next_step()
 
     assert mocked_post.call_count == 0
+
+
+def test_post_to_studio_include_prefix_if_needed(tmp_dir, mocker, monkeypatch):
+    dvc_repo = mocker.MagicMock()
+    dvc_repo.scm.get_rev.return_value = "f" * 40
+    dvc_repo.index.stages = []
+    dvc_repo.scm.get_ref.return_value = None
+    mocker.patch("dvclive.live.get_dvc_repo", return_value=dvc_repo)
+
+    mocked_response = mocker.MagicMock()
+    mocked_response.status_code = 200
+    mocked_post = mocker.patch("requests.post", return_value=mocked_response)
+
+    monkeypatch.setenv(STUDIO_ENDPOINT, "https://0.0.0.0")
+    monkeypatch.setenv(STUDIO_REPO_URL, "STUDIO_REPO_URL")
```

Comment:
The change in https://github.com/iterative/dvc-studio-client/issues/4 should be transparent here and should not require any changes.

In the tests I am just patching the env var for convenience instead of having to patch `dvc-studio-client`

Reply:
So let's not forget to get rid of it completely after the `dvc-studio-client` bump